src/machine/spi.go

```diff
@@ -1,4 +1,4 @@
-// +build !baremetal sam stm32,!stm32f7x2,!stm32l5x2 fe310 k210 atmega
+// +build !baremetal atsamd21 stm32,!stm32f7x2,!stm32l5x2 fe310 k210 atmega
```

Comment:
Should this be `atsamd51` in this PR?

Reply:
`SPI.Tx()` in samd51 is now defined in `machine_atsamd51.go` j.
So I think `atsamd21` or `sam,!atsamd51` is correct for this part.

However, after this PR is merged, the samd21 version will be improved, so this line will look like this
So I think either way is fine.

```
// +build !baremetal stm32,!stm32f7x2,!stm32l5x2 fe310 k210 atmega
```

Reply:
OK got it.

Does that mean you want to change to

```go
// +build !baremetal stm32,!stm32f7x2,!stm32l5x2 fe310 k210 atmega
```

as a part of this PR?

Reply:
I will fix it in the next PR I launch.
So, the current source code is fine for this PR.